xml/System/Environment.xml

```diff
@@ -2493,21 +2493,21 @@ The following example creates environment variables for the <xref:System.Environ
         <ReturnType>System.Version</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a version consisting of the major, minor, build, and revision numbers of the common language runtime.</summary>
-        <value>The version of the common language runtime.</value>
+        <summary>Gets a version consisting of the major, minor, build, and revision numbers of the Common Language Runtime.</summary>
+        <value>The version of the Common Language Runtime.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- For the .NET Framework Versions 4, 4.5, 4.5.1, and 4.5.2, the <xref:System.Environment.Version%2A?displayProperty=nameWithType> property returns a <xref:System.Version> object whose string representation has the form `4.0.30319.xxxxx`. For the .NET Framework 4.6 and later versions, it has the form `4.0.30319.42000`.
+ For the .NET Framework Versions 4, 4.5, 4.5.1, and 4.5.2, the <xref:System.Environment.Version%2A?displayProperty=nameWithType> property returns a <xref:System.Version> object whose string representation has the form `4.0.30319.xxxxx`. For the .NET Framework 4.6 and later versions, and .NET Core versions before 3.0, it has the form `4.0.30319.42000`. For .NET Core 3.0 and later versions, this property returns the actual .NET Core version number.
```

Comment:
```suggestion
  In .NET Core 3.x and .NET 5+, the <xref:System.Environment.Version%2A?displayProperty=nameWithType> property returns .NET runtime version number.

  For the .NET Framework Versions 4, 4.5, 4.5.1, and 4.5.2, the <xref:System.Environment.Version%2A?displayProperty=nameWithType> property returns a <xref:System.Version> object whose string representation has the form `4.0.30319.xxxxx`. For the .NET Framework 4.6 and later versions, and .NET Core versions before 3.0, it has the form `4.0.30319.42000`.
```
We should put the simple forward looking description first, and the description of the non-sensical past second.